test/gtest/unit_conv_solver_ConvOclBwdWrW2NonTunable.cpp

```diff
@@ -42,7 +44,12 @@ auto GetConvTestCases(miopenDataType_t datatype)
 const auto& GetTestParams()
 {
     static const auto params = [] {
-        auto p = miopen::unit_tests::UnitTestConvSolverParams(Gpu::All);
+#if WORKAROUND_MIOPEN_1392
```

Comment:
```suggestion
#if WORKAROUND_LWPMIOPEN_1392
```

See the first comment